zookeeper-server/src/test/java/org/apache/zookeeper/common/AtomicFileWritingIdiomTest.java

```diff
@@ -348,12 +350,12 @@ private String getContent(File file, String encoding) throws IOException {
     }
 
     private String getContent(File file) throws IOException {
-        return getContent(file, "ASCII");
+        return getContent(file, StandardCharsets.US_ASCII);
```

Comment:
Do we use ASCII anywhere else?

Reply:
For this test class, it's consistent in that it expects Ascii on line 356/358.  Ascii fits perfectly fine into UTF-8 so this test is not invalid and probably doesn't add any value to change it because the data being used in this test is not outside the ASCII range.